inc/classes/class-srm-post-type.php

```diff
@@ -562,6 +575,11 @@ public function redirect_rule_metabox( $post ) {
 		$enable_regex   = get_post_meta( $post->ID, '_redirect_rule_from_regex', true );
 
 		if ( empty( $status_code ) ) {
+			/**
+			 * Filter the default status code to redirect with
+			 *
+			 * @return int
```

Comment:
```suggestion
			 * Filter the default HTTP status code to redirect with.
			 *
			 * Which HTTP redirect code safe redirect manager should default to. This can
			 * be overridden in the dashboard for each redirect.
			 *
			 * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections
			 *
			 * @param int $default_status_code Default redirect statues. Default value `302`.
```